README.md

```diff
@@ -2,100 +2,128 @@
 
 [![Build Status](https://secure.travis-ci.org/jzaefferer/commitplease.png)](http://travis-ci.org/jzaefferer/commitplease)
 
-This [node.js](http://nodejs.org/) module validates git commit messages while you commit, according to the rules specified in the [jQuery Commit Guidelines](http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines). These are pretty generic, so this module should be useful for various projects that care about commit message formats.
+This [node.js](http://nodejs.org/) module makes sure your commit
+messages consistently follow one of these style guides:
+ 1. [jQuery Commit Guidelines][1]
+ 2. [AngularJS Commit Guidelines][2]
+ 3. Your own (if you really feel like it)
```

Comment:
Lets drop the third item and add something like "You can also start with one of these and customize the validation rules."